cmake/features.cmake

```diff
@@ -10,6 +10,8 @@ ie_dependent_option (ENABLE_INTEL_CPU "CPU plugin for OpenVINO Runtime" ON "RISC
 
 ie_dependent_option (ENABLE_ARM_COMPUTE_CMAKE "Enable ARM Compute build via cmake" OFF "ENABLE_INTEL_CPU" OFF)
 
+ie_dependent_option (ENABLE_LLMDNN "Enable library for LLM in CPU plugin, need gcc 11.3+" OFF "ENABLE_INTEL_CPU AND X86_64 AND LINUX" OFF)
```

Comment:
should we enable it for Windows? I see that cmake scripts handle MSVC
What is gcc 11.3 specific here?

We don't use such new compiler for OpenVINO and it means that depending on the system users will see different performance.

Reply:
Support AMX windows will be delayed, the script is just removed the icc dependency, the function is not verified.
GCC 11 begins to support AMX instruction which is needed by llmdnn library. This library is focused on AMX which can help LLM a lot, but it needs a newer compiler, gcc 11 at least.
The 'need gcc 11.3...' is removed from the description, when compiling the library will give the detail error message if version is not correct.

Reply:
But we don't build OpenVINO with gcc 11, does it mean this feature will be always OFF ?